### ISSUE TITLE
Windows - args.gn doesn't get created correctly

### ISSUE THREAD

**Issue by azdps** [Eloston/ungoogled-chromium#604]
Windows ungoogled-master branch 11/26/2018.

Windows bug [#583](https://github.com/Eloston/ungoogled-chromium/issues/583) that was suppose to be fixed with commit [6987eca](https://github.com/Eloston/ungoogled-chromium/commit/6987eca9ac2d6b7516702f0afcad63b3a16eef84) didn't work as expected.

I'm attaching the args.gn file that it just produced for me during the windows build process. The new line endings are being placed after each word.

See the attached file. The code in question is from the build.py file. I changed the gn extension to txt since github won't accept the gn extension for attachments. [args.txt](https://github.com/Eloston/ungoogled-chromium/files/2617483/args.txt)

`(source_tree / 'out/Default/args.gn').write_text('\n'.join(str(bundle.gn_flags)), encoding=ENCODING)`

**Comment by azdps** [comment]
Just for comparison the old code produced the following file: [args.txt](https://github.com/Eloston/ungoogled-chromium/files/2617511/args.txt)

This is the old code prior to changing it:
`(source_tree / 'out/Default/args.gn').write_text('\n'.join(bundle.gn_flags), encoding=ENCODING)`

**Comment by azdps** [comment]
The is what the args.gn file should look like. I was able to print the information using the command:
`print(str(bundle.gn_flags), end='')`

I'm just not familiar enough with python to fix this issue. I do understand that the ungoogled-chromium Windows build process is in need of update. I think this will at least help the build process.

```
blink_symbol_level=0
chrome_pgo_phase=0
clang_use_chrome_plugins=false
enable_ac3_eac3_audio_demuxing=true
enable_hangout_services_extension=false
enable_hevc_demuxing=true
enable_iterator_debugging=false
enable_mdns=false
enable_mse_mpeg2ts_stream_parser=true
enable_nacl=false
enable_nacl_nonsfi=false
enable_one_click_signin=false
enable_reading_list=false
enable_remoting=false
enable_reporting=false
enable_service_discovery=false
enable_swiftshader=false
enable_widevine=true
exclude_unwind_tables=true
fatal_linker_warnings=false
ffmpeg_branding="Chrome"
fieldtrial_testing_like_official_build=true
full_wpo_on_official=false
google_api_key=""
google_default_client_id=""
google_default_client_secret=""
is_clang=true
is_debug=false
is_official_build=true
optimize_webui=false
proprietary_codecs=true
rtc_use_lto=false
safe_browsing_mode=0
symbol_level=0
target_cpu="x64"
treat_warnings_as_errors=false
use_gnome_keyring=false
use_jumbo_build=true
use_official_google_api_keys=false
use_ozone=false
use_sysroot=false
use_unofficial_version_number=false
```

**Comment by Eloston** [comment]
Resolved by #607.
